docs/data_formats.md

```diff
@@ -25,6 +27,8 @@ In particular, the following file formats are supported:
 - Single-file images (tif, czi, nifti, raw)
 - KNOSSOS file hierarchy 
 
+Note, for datasets in the Zarr, N5, Neuroglancer `Pre-Computed` or BossDB formats uploading and automatic conversion are not supported. Rather, directly stream them from an HTTP server or the cloud. See the page on [datasets](./datasets.md) for uploading and configuring these formats.
```

Comment:
```suggestion
Note, for datasets in the Zarr, N5, Neuroglancer `Pre-Computed` or BossDB formats uploading and automatic conversion are not supported.
Instead, they can be directly streamed from an HTTP server or the cloud.
See the page on [datasets](./datasets.md) for uploading and configuring these formats.
```

i changed the sentence about the streaming a bit.

by the way, I prefer one sentence per line (the line breaks are not rendered by markdown anyway) for easier reviewing, diffing and inspecting the length of a sentence.